google-cloud-datastore/src/main/java/com/google/cloud/datastore/aggregation/CountAggregation.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.datastore.aggregation;
+
+import com.google.datastore.v1.AggregationQuery;
+import com.google.datastore.v1.AggregationQuery.Aggregation.Count;
+import com.google.protobuf.Int64Value;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Consumer;
+
+public class CountAggregation extends Aggregation {
+
+  private final long limit;
+
+  private CountAggregation(String alias, long limit) {
```

Comment:
that make sense @gforgurups . There is nothing wrong in giving creational access to Aggregations. I'll make this change and add one more version `addAggregation(Aggregation)`method along with `addAggregation(AggregationBuilder)`. so that user can pass Aggregation directly without it's builder.

Reply:
Fixed as part of https://github.com/googleapis/java-datastore/pull/823/commits/5097cd69fc6c1684103fa4500c289b86fe81d1ce.